setup_configure.py

```diff
@@ -58,6 +58,7 @@ class EnvironmentOptions(object):
     def __init__(self):
         self.hdf5 = os.environ.get('HDF5_DIR')
         self.hdf5_version = os.environ.get('HDF5_VERSION')
+        self.mpi = os.environ.get('HDF5_PARALLEL') == "ON"
```

Comment:
Could this be done in a 'truthy' way that does not privilege the string "ON"?  I would expect `HDF5_PARALLEL=1` to also work.